daycare/test_daycare.py

```diff
@@ -61,3 +64,23 @@ def test_daycare_user_creation_with_same_user_name(self, create_daycare_user):
             DayCare.create(username='testUser01', email="valid@gmail.com", password='pass123', name='daycare',
                            description='new description', price_per_day=10, capacity=50,
                            area='north', city='haifa', address='new address')
+
+    def test_daycare_redirected_homepage_after_login(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get('/')
+        assert response.status_code == 302
+        assert response['Location'] == '/homepage/'
+
+    def test_relevant_daycare_homepage_with_static_data(self, client, create_daycare_user):
+        dayCare = random.choice([daycare for daycare in DayCare.objects.all() if 'static' in daycare.user.username])
+        client.force_login(user=dayCare.user)
+
+        response = client.get('/homepage/')
+
+        daycare_shown_in_homepage = response.context['daycare']
+        images_shown_in_homepage = set(response.context['images'])
+        reviews_shown_in_homepage = set(response.context['reviews'])
```

Comment:
better to have it like this:
```
daycare_shown_in_homepage = response.context['daycare']
assert daycare_shown_in_homepage == dayCare

images_shown_in_homepage = set(response.context['images'])
assert images_shown_in_homepage == set(Image.get_images_by_daycare_id(daycare_id=dayCare.id))
```

I'd also put into a variable the other side of the equation.

Reply:
Fixed